index.html

```diff
@@ -80,7 +100,6 @@ <h3>I'm a software developer</h3>
     </section>
   </div>
 
-
   <!-- Works Section -->
   <section class="works">
     <h2>My Recent Works</h2>
```

Comment:
- [ ] Please edit the spacing of the "works" section on the mobile view to match the Figma design. This section is taking more width than it's supposed to take which breaks the responsivity of the other section. (See image below).

![image](https://user-images.githubusercontent.com/62290545/159705062-cb6c190d-0e8c-4688-903b-18c6d7744f2d.png)

- This is likely caused by too much padding and margin on the list of technologies in each project card.